utils/verify_intel_gaudi/roles/verify_intel_gaudi/tasks/main.yml

```diff
@@ -33,7 +33,7 @@
   when: lspci_output.stdout | length > 0
 
 - name: Include omnia_config.yml
-  ansible.builtin.include_vars: "{{ role_path }}/../../../input/omnia_config.yml"
+  ansible.builtin.include_vars: "{{ role_path }}/../../../../../input/omnia_config.yml"
```

Comment:
This can be deleted now. The purpose to include omnia_config.yml is to get the value of verify_intel_gaudi_with_sanity_test_only.

Reply:
Done